scheduler/test/cook/test/mesos/util.clj

```diff
@@ -393,21 +393,50 @@
             _ (Thread/sleep 5)
             end-time (Date.)
             states [(name state)]
-            name (constantly true)]
+            namet (constantly true)
+            namef (constantly false)
```

Comment:
Maybe `match-any-name` and `match-no-name`?

Reply:
Renamed to match-any-name-fn and match-no-name-fn.